### PR TITLE
Update Cypress test for case 6.

### PR DESCRIPTION
**Pull request by dvkruchinin**
Dynamically retrieving values for calculating the correctness of scale.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/case_6_image_scale_fit.js [hunk 1]
```diff
@@ -32,17 +34,24 @@ context('Check if the image is scaled and then fitted', () => {
     describe(`Testing "${labelName}"`, () => {
         it('Scale image', () => {
             cy.get('#cvat_canvas_background')
-                .should('have.attr', 'style').and('contain', 'scale(1.05)')
+            .should('have.attr', 'style')
+            .then($scale => {
```
Comment (bsekachev) [inline tests/cypress/integration/case_6_image_scale_fit.js:38]: I suppose it isn't scale, it's ``styles``

File: tests/cypress/integration/case_6_image_scale_fit.js [hunk 2]
```diff
@@ -32,17 +34,24 @@ context('Check if the image is scaled and then fitted', () => {
...
+                scaleBefore = Number($scale.match(/scale\((\d\.\d+)\)/m)[1])
+            })
             cy.get('.cvat-canvas-container')
-                .trigger('wheel', {deltaY: 5})
+            .trigger('wheel', {deltaY: 5})
             cy.get('#cvat_canvas_background')
-                .should('have.attr', 'style').and('contain', 'scale(0.875)')
+            .should('have.attr', 'style')
+            .then($scale => {
```
Comment (bsekachev) [inline tests/cypress/integration/case_6_image_scale_fit.js:45]: The same

File: tests/cypress/integration/case_6_image_scale_fit.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/case_6_image_scale_fit.js:38]: This is getting all the parameters of the style attribute.
For example:
```
top: 20px; left: 73px; width: 800px; height: 800px; filter: brightness(1) contrast(1) saturate(1); transform: scale(1.05) rotate(0deg);
```

 Then use the regular expression to get the scale value.
For example:
```
1.05
```

File: tests/cypress/integration/case_6_image_scale_fit.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/integration/case_6_image_scale_fit.js:45]: Answered above

File: tests/cypress/integration/case_6_image_scale_fit.js (hunk 1 above)
Comment (bsekachev) [reply tests/cypress/integration/case_6_image_scale_fit.js:38]: Yes, I mean that the parameter of then() - ``scale`` is not actual scale, it's all the styles where you find scale with regex.

Comment (bsekachev) [reply tests/cypress/integration/case_6_image_scale_fit.js:38]: The issue about naming

Comment (dvkruchinin) [reply tests/cypress/integration/case_6_image_scale_fit.js]: I understood. Thanks. Corrected.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**